src/driver/communication_interface.h

```diff
@@ -194,6 +200,9 @@ class CommunicationInterface {
   void PublishPlanComplete(const int64_t& plan_utime, bool success = true,
                            std::string driver_status_string = "");
 
+  // set driver status
```

Comment:
not in this PR but 3 line above in L200, fundamental types should be passed by value. also add missing const and `&`